.github/workflows/release-orchestrate-overall.yml

```diff
@@ -7,82 +7,127 @@ on:
-
-          # want >=5
-          bash --version
+    - uses: che-incubator/trigger-workflow-action@main
+      with:
+        repo: eclipse/che-theia
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        workflow_id: "5717988"
+        workflow_name: "Release Che Theia"
+        parameters: "{\"version\":\${{ github.event.inputs.version }}"\"}"
+        wait_interval: "120"
+        wait_initial_delay: "20"
+        
+  release-che-machine-exec:
+    steps:
+    - uses: che-incubator/trigger-workflow-action@main
+      with:
+        repo: eclipse/che-machine-exec
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        workflow_id: "5149341"
+        workflow_name: "Release Che Machine Exec"
+        parameters: "{\"version\":\${{ github.event.inputs.version }}"\"}"
+        wait_interval: "120"
+        wait_initial_delay: "20"
+        
+        
+  release-che-jwtproxy:
+    steps:
+    - uses: che-incubator/trigger-workflow-action@main
+      with:
+        repo: eclipse/che-machine-exec
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        workflow_id: "5149341"
+        workflow_name: "Release Che Machine Exec"
+        parameters: "{\"branch\":\${{ github.event.inputs.version }}"\"}"
+        wait_interval: "120"
+        wait_initial_delay: "5"
 
-          # do we need any of these?
-          # docker-ce          x86_64  3:19.03.14-3.el7            docker-ce-stable   24 M
-          # gcc-c++            x86_64  4.8.5-44.el7                base              7.2 M
-          # nodejs             x86_64  2:10.23.0-1nodesource       nodesource         20 M
-          # yarn               noarch  1.22.5-1                    yarn              1.2 M
-      - name: Release
-        run: |
-          CHE_VERSION="${{ github.event.inputs.version }}"
-          echo "CHE_VERSION=${CHE_VERSION}"
-          sed -i VERSION -r -e "s#CHE_VERSION=.*#CHE_VERSION=${CHE_VERSION}#"
-          PHASES="${{ github.event.inputs.phases }}"
-          echo "PHASES=${PHASES}"
-          sed -i VERSION -r -e "s#PHASES=.*#PHASES=${PHASES}#"
+  release-kubernetes-image-puller:
+    steps:
+    - uses: che-incubator/trigger-workflow-action@main
+      with:
+        repo: eclipse/che-machine-exec
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        workflow_id: "5149341"
+        workflow_name: "Release Che Machine Exec"
+        parameters: "{\"branch\":\${{ github.event.inputs.version }}"\"}"
+        wait_interval: "120"
+        wait_initial_delay: "5"
+        
+  release-che-devfile-regisrty:
+    steps:
+    - uses: che-incubator/trigger-workflow-action@main
+      with:
+        repo: eclipse/che-devfile-registry
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        workflow_id: "5149341"
+        workflow_name: "Release Che Devfile Registry"
+        parameters: "{\"branch\":\${{ github.event.inputs.version }}"\"}"
+        wait_interval: "120"
+        wait_initial_delay: "30"
```

Comment:
if we know that these steps will take at minimum 30 mins, then this is a good hack to avoid needless api calls. OTOH if the job fails fast, we want to know. So maybe we set this to 5 mins initially, and if it 's consistently successful we can increase the initial delay in future?